torch/csrc/utils/python_numbers.h

```diff
@@ -59,6 +63,18 @@ inline int64_t THPUtils_unpackLong(PyObject* obj) {
   return (int64_t)value;
 }
 
+inline int64_t THPUtils_unpackBool(PyObject* obj) {
+  int overflow;
```

Comment:
where is this used?  I.e. why did you use this instead of just checking if you have Py_TRUE or Py_False?  This is always called after checkBool, right so you only have those values?

Also I'm not sure if this will work, because in python 2 bools are an instance of int, not long.

Reply:
this method was removed in the very last iteration.